### PR TITLE
generate dummy buffer instead of using a random mp3

### PR DESCRIPTION
**Pull request by BaconPancakes**

### CODE REVIEW

File: client/src/components/AudioDisplayTable/AudioDisplayTable.jsx [hunk 1]
```diff
@@ -125,40 +126,59 @@ class AudioDisplayTable extends Component {
...
             isExpanded: newIsExpanded
-        })
-    };
+        });
+    }
 
     render() {
         const { classes } = this.props;
-
+        let baselinePlayer = this.state.baselinePlayerSet ? this.props.game.baselinePlayer : null;
```
Comment (shannonphu) [inline client/src/components/AudioDisplayTable/AudioDisplayTable.jsx]: isnt this.props.game.baselinePlayer null anyways if it wasnt set?

File: client/src/components/AudioWaveform/AudioWaveform.jsx [hunk 2]
```diff
@@ -25,44 +26,89 @@ class AudioWaveform extends Component {
...
+                webAudio: {
+                    audioBuffer: myArrayBuffer,
+                    scale: 128,
+                    multiChannel: false
+                },
+                keyboard: true,
+                showPlayheadTime: true,
+                zoomLevels: [128, 256, 512, 1024, 2048, 4096]
+            };
+
```
Comment (shannonphu) [inline client/src/components/AudioWaveform/AudioWaveform.jsx:62]: This and peaks.init is same code as in the else block? Can you just pull both outside of the if statement

File: client/src/components/AudioWaveform/Player.js [hunk 3]
```diff
@@ -15,14 +15,14 @@ export function Player(audioBuffer) {
...
-                }
-            }, 0.25);
+            // Tone.Transport.scheduleRepeat(() => {
+            //     var time = this.getCurrentTime();
+            //     eventEmitter.emit('player.timeupdate', time);
+
+                // if (time >= this.getDuration()) {
+                //     Tone.Transport.stop();
+                // }
+            // }, 0.25);
```
Comment (shannonphu) [inline client/src/components/AudioWaveform/Player.js:25]: remove

File: client/src/components/AudioWaveform/Player.js [hunk 4]
```diff
@@ -50,7 +50,7 @@ export function Player(audioBuffer) {
         },
 
         seek: function(time) {
-            Tone.Transport.seconds = time;
+            // Tone.Transport.seconds = time;
```
Comment (shannonphu) [inline client/src/components/AudioWaveform/Player.js:53]: rm

**Review by shannonphu — CHANGES_REQUESTED** [review]

File: client/src/components/AudioDisplayTable/AudioDisplayTable.jsx (hunk 1 above)
Comment (BaconPancakes) [reply client/src/components/AudioDisplayTable/AudioDisplayTable.jsx]: probs

**Comment by BaconPancakes** [comment]
> This and peaks.init is same code as in the else block? Can you just pull both outside of the if statement 

The else block has a bunch of promises for fetching audioUrl, whereas the if block is generating an audiobuffer outside of the fetch

**Comment by BaconPancakes** [comment]
> rm 

want to keep it in case that functionality breaks something i havent tested

**Pull request merged**